observability/distr_tracing/distr_tracing_rn/distr-tracing-rn-3-2.adoc

```diff
@@ -0,0 +1,129 @@
+
+[id="distributed-tracing-rn_3-2_tempo-release-notes_new-features-and-enhancements"]
+=== New features and enhancements
+
+This update introduces the following enhancements:
+
+* {TempoName} 3.2 is based on the open source link:https://grafana.com/oss/tempo/[Grafana Tempo] 2.4.1.
+* Allowing the overriding of resources per component.
+
+////
+[id="distributed-tracing-rn_3-2_tempo-release-notes_deprecated-functionality"]
+=== Deprecated functionality
+
+In the {TempoName} 3.2, ???
+////
+
+////
+[id="distributed-tracing-rn_3-2_tempo-release-notes_removal-notice"]
+=== Removal notice
+
+In {TempoName} 3.2, the FEATURE has been removed. Bug fixes and support are provided only through the end of the 3.? lifecycle. As an alternative to the FEATURE for USE CASE, you can use the ALTERNATIVE instead.
+////
+
+[id="distributed-tracing-rn_3-2_tempo-release-notes_bug-fixes"]
+=== Bug fixes
+
+This update introduces the following bug fixes:
+
+* Before this update, the Jaeger UI only displayed services that had sent traces in the previous 15 minutes. With this update, the availability of the service and operation names can be configured by using the following field: `spec.template.queryFrontend.jaegerQuery.servicesQueryDuration`. (link:https://issues.redhat.com/browse/TRACING-3139[TRACING-3139])
+* Before this update, the `query-frontend` pod might get stopped when out-of-memory (OOM) as a result of searching a large trace. With this update, resource limits can be set to prevent this issue. (link:https://issues.redhat.com/browse/TRACING-4009[TRACING-4009])
+
+[id="distributed-tracing-rn_3-2_tempo-release-notes_known-issues"]
+=== Known issues
+
+There is currently a known issue:
+
+* Currently, the {TempoShortName} fails on the IBM Z (`s390x`) architecture. (link:https://issues.redhat.com/browse/TRACING-3545[TRACING-3545])
+
+[id="distributed-tracing-rn_3-2_jaeger-release-notes"]
+== {JaegerName}
+
+The {JaegerName} is provided through the {JaegerOperator} Operator.
+
+[IMPORTANT]
+====
+Jaeger does not use FIPS validated cryptographic modules.
+====
+
+[id="distributed-tracing-rn_3-2_jaeger-release-notes_support-for-elasticsearch-operator"]
+=== Support for {es-op}
+
+{JaegerName} 3.2 is supported for use with the {es-op} 5.6, 5.7, and 5.8.
+
+[id="distributed-tracing-rn_3-2_jaeger-release-notes_deprecated-functionality"]
+=== Deprecated functionality
+
+In the {DTProductName} 3.2, Jaeger and support for Elasticsearch remain deprecated, and both are planned to be removed in a future release.
+Red Hat will provide critical and above CVE bug fixes and support for these components during the current release lifecycle, but these components will no longer receive feature enhancements.
+The {TempoOperator} and the {OTELName} are the preferred Operators for distributed tracing collection and storage.
+The OpenTelemetry and Tempo distributed tracing stack is to be adopted by all users because this will be the stack that will be enhanced going forward.
```

Comment:
```suggestion
Broad adoption of the OpenTelemetry and Tempo distributed tracing stack by all users is expected because this stack that will be enhanced going forward.
```

Reply:
A suggestion to avoid future tense, per https://www.ibm.com/docs/en/ibm-style?topic=grammar-verbs#tense